### PR TITLE
ref(py3): Fix email debug tests

### PR DESCRIPTION
**Pull request by evanpurkhiser**
I've created a new set of fixtures for the python 2 version of these tests, and updated the original fixtures to the python3 results.

It's explained in the comment I've added in the test why we have to do this.

### CODE REVIEW

File: tests/acceptance/test_emails.py [hunk 1]
```diff
@@ -36,9 +37,17 @@
...
+    # NOTE that we _cannot_ just set the version of the seed, as there are more
+    # differences. See [0].
+    #
+    # [0]: https://stackoverflow.com/questions/55647936/random-randint-shows-different-output-in-python-2-x-and-python-3-x-with-same-see/55648073
+    version_suffix = "_py2" if six.PY2 else ""
+
     # "sso unlinked without password"
     # => "sso_unlinked_without_password.txt"
-    filename = name.replace(" ", "_") + ".txt"
+    filename = name.replace(" ", "_") + version_suffix + ".txt"
```
Comment (wedamija) [inline tests/acceptance/test_emails.py:50]: +1 to naming the py3 things as the default name.

**Review by wedamija — APPROVED** [review]

**Comment by joshuarli** [comment]
Oh, I never did sweeping changes on non-python files before... nice.

File: tests/acceptance/test_emails.py [hunk 2]
```diff
@@ -36,9 +37,17 @@
 
 
 def read_txt_email_fixture(name):
+    # XXX(python3): We have different fixtures for python2 vs python3 tests
```
Comment (BYK) [inline tests/acceptance/test_emails.py:40]: We can easily replace this `random` and `seed` logic with `uuid` and keep getting consistent "random" numbers across python versions (and probably even across languages):

```
byk@DESKTOP-QII4CIG:$ python
Python 2.7.17 (default, Nov  7 2019, 10:07:09)
[GCC 7.4.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import uuid
>>> uuid.UUID(bytes=b'abcdabcdabcdabc1', version=4)
UUID('61626364-6162-4364-a162-636461626331')
>>> uuid.UUID(bytes=b'abcdabcdabcdabc1', version=4)
UUID('61626364-6162-4364-a162-636461626331')
>>> uuid.UUID(bytes=b'abcdabcdabcdabc1', version=4)
UUID('61626364-6162-4364-a162-636461626331')
>>> uuid.UUID(bytes=b'abcdabcdabcdabc2', version=4)
UUID('61626364-6162-4364-a162-636461626332')
byk@DESKTOP-QII4CIG:$ logout
[byk@DESKTOP-QII4CIG ~/P/g/sentry]$ python                                                                                                        Python 3.8.6rc1 (tags/v3.8.6rc1:08bd63d, Sep  7 2020, 23:10:23) [MSC v.1927 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import uuid
>>> uuid.UUID(bytes=b'abcdabcdabcdabc1', version=4)
UUID('61626364-6162-4364-a162-636461626331')
>>> uuid.UUID(bytes=b'abcdabcdabcdabc2', version=4)
UUID('61626364-6162-4364-a162-636461626332')
```

Comment (BYK) [reply tests/acceptance/test_emails.py:40]: Turns out we have an easier way out if this works. `random.seed()` has a new `version` parameter which should produce the same values with the PY2 implementation when `version=1`. So something like this should fix this:

```py
def get_random(request):
    seed = request.GET.get("seed", six.text_type(time.time()))
    custom_random = Random()
    if six.PY3:
        custom_random.seed(seed, version=1)
    else:
        custom_random.seed(seed)
    return custom_random
```

Comment (BYK) [reply tests/acceptance/test_emails.py:40]: Okay, ignore what I said, turns out there's some PhD-level `random` usage here so I give up :D

**Pull request merged**